python/lsst/ts/ATDome/dome_csc.py

```diff
@@ -177,74 +210,103 @@ def __init__(self, config_dir=None, initial_state=salobj.State.STANDBY,
         self.assert_enabled("homeAzimuth")
         if self.evt_azimuthState.data.homing:
             raise salobj.ExpectedError("Already homing")
-        self.evt_azimuthCommandedState.set_put(commandedState=AzimuthCommandedState.HOME,
-                                               azimuth=math.nan, force_output=True)
+        self.evt_azimuthCommandedState.set_put(
+            commandedState=AzimuthCommandedState.HOME,
+            azimuth=math.nan,
+            force_output=True,
+        )
         await self.run_command("HM")
         self.status_sleep_task.cancel()
 
     async def do_moveShutterDropoutDoor(self, data):
         """Implement the ``moveShutterDropoutDoor`` command."""
         self.assert_enabled("moveShutterDropoutDoor")
         if self.evt_mainDoorState.data.state != ShutterDoorState.OPENED:
-            raise salobj.ExpectedError("Cannot move the dropout door until the main door is fully open.")
+            raise salobj.ExpectedError(
+                "Cannot move the dropout door until the main door is fully open."
```

Comment:
Not true, it just has to be opened far enough... which is when the "interlock" switch deactivates... which is something like 10 degrees away from collision or something.

Reply:
The low level controller does not provide the reading of the interlock switch so the only safe thing for the CSC to do is require the main door to be fully open.

Reply:
I suppose we could try just sending the command and seeing what the controller does. But I'd hate to find out the hard way that we end up crashing the doors together.

Reply:
At the moment I'm testing remotely and I don't want to test this without someone there... so lets leave this for another change.
I'll consult the vendor regarding the issues above.